src/test/java/com/philvarner/clamavj/test/ScanResultTestCase.java

```diff
@@ -0,0 +1,26 @@
+package com.philvarner.clamavj.test;
+
+import com.philvarner.clamavj.ScanResult;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class ScanResultTestCase {
+
+	/**
+	 * Detect scan-failures correctly, as reported by ClamAV.
+	 * 
+	 * ClamAV reports errors by using a suffix of "ERROR". See
+	 * https://github.com/vrtadmin/clamav-devel/blob/79efd61b432f71b3f420d2582c352f15a81099cf/clamd/session.c#L169
+	 * 
+	 * To reproduce, set <code>TemporaryDirectory</code> in clamd.conf to
```

Comment:
is the temp dir set somewhere in our setup? i don't see anything in our clamd.conf

Reply:
Which one, production, vm, or in this project?